cigaradvisor/blocks/imagecta/imagecta.css

```diff
@@ -23,9 +23,16 @@
 
 .section[data-layout="50/50"] .imagecta-wrapper{
   display: flex;
+}
+
+.section[data-layout="50/50"].article-teaser-container .imagecta-wrapper{
   flex-basis: 100%;
 }
 
+.section.imagecta-container.separator-container:not([class*=""]) .layout-wrapper > div {
```

Comment:
Can you get me an example of what this is trying to fix? Maybe i'm not understanding the need?

Reply:
Sure, when the number of imageCTAs are not same on left and right grids of a 2-col layout, imageCTA are getting stretched to occuppy the available space..which is not how it looks on the live site.
![image](https://github.com/hlxsites/famous-smoke-cigaradvisor/assets/53608019/e29c1ab6-852a-444c-8f98-a0d3019e5185)